src/Chorus/UI/Misc/ServerSettingsControl.cs

```diff
@@ -20,11 +20,7 @@ public ServerSettingsControl()
 		{
 			InitializeComponent();
 
-			_serverLabel.Visible = ServerSettingsModel.IsQaServer || ServerSettingsModel.IsPrivateServer;
-			if (_serverLabel.Visible)
-			{
-				_serverLabel.Text = string.Format(_serverLabel.Text, $"{(ServerSettingsModel.IsPrivateServer ? "private" : null)}{ServerSettingsModel.LanguageForgeServer}");
-			}
+			_serverLabel.Text = string.Format(_serverLabel.Text, $"{(ServerSettingsModel.IsPrivateServer ? "private" : "public")}{ServerSettingsModel.LanguageForgeServer}");
```

Comment:
A comment with the (typical) string we end up with might be helpful.

Reply:
Would `// format the string with the server URL. We build our own URL because 'resumable' is not helpful here.` suffice?